src/main/java/seedu/address/logic/commands/ResumePreviewCommand.java

```diff
@@ -99,10 +87,36 @@ private String getDataFromResume(Resume resume, Model model) {
         for (String skill: skillsList) {
             data.append(skill).append("\n");
         }
-
         return data.toString();
     }
 
+    private List<String> getStringListOfInternships(Resume resume, Model model) {
+        return resume.getInternshipIds()
+                .stream()
+                .map(x -> model.hasInternshipId(x) ? model.getInternshipById(x) : null)
+                .filter(Objects::nonNull)
+                .map(Internship::toPreview)
+                .collect(Collectors.toList());
+    }
+
+    private List<String> getStringListOfProjects(Resume resume, Model model) {
+        return resume.getProjectIds()
+                .stream()
+                .map(x -> model.hasProjectId(x) ? model.getProjectById(x) : null)
+                .filter(Objects::nonNull)
+                .map(Project::toPreview)
+                .collect(Collectors.toList());
+    }
+
+    private List<String> getStringListOfSkills(Resume resume, Model model) {
+        return resume.getSkillIds()
+                .stream()
+                .map(x -> model.hasSkillId(x) ? model.getSkillById(x) : null)
+                .filter(Objects::nonNull)
+                .map(Skill::toPreview)
+                .collect(Collectors.toList());
+    }
+
```

Comment:
Can you try running checkstyleMain locally? When I did this, I got scolded to write Javadocs for them :'(

Reply:
My checkstyle runs fine, maybe it's cause the function is short?

Reply:
Hmm perhaps. All right then if checkstyleMain doesn't flag anything!